lib/fog/billing/storm_on_demand.rb

```diff
@@ -1,10 +1,10 @@
 module Fog
   module Billing
     class StormOnDemand < Fog::Service
-      autoload :Invoice, 'fog/billing/storm_on_demand/models/invoice'
-      autoload :Invoices, 'fog/billing/storm_on_demand/models/invoices'
-      autoload :Payment, 'fog/billing/storm_on_demand/models/payment'
-      autoload :Payments, 'fog/billing/storm_on_demand/models/payments'
+      autoload :Invoice, File.expand_path('../billing/storm_on_demand/models/invoice', __FILE__)
+      autoload :Invoices, File.expand_path('../billing/storm_on_demand/models/invoices', __FILE__)
+      autoload :Payment, File.expand_path('../billing/storm_on_demand/models/payment', __FILE__)
+      autoload :Payments, File.expand_path('../billing/storm_on_demand/models/payments', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.